neurots/schemas/distributions.json

```diff
@@ -49,6 +49,12 @@
                     "$ref": "#/definitions/data_distr",
                     "description": "The data distribution used for the number of trees to synthesize"
                 },
+                "min_bar_length": {
+                    "type": "number",
+                    "minimum": 0,
+                    "minimum": 0,
```

Comment:
I think one minimum is enough @arnaudon ;-p

Reply:
ah lol

Reply:
:P